cel/README.md

```diff
@@ -37,33 +42,34 @@ kubectl apply --filename https://storage.cloud.google.com/tekton-releases-nightl
 
 ## Usage
 
-To evaluate a CEL expressions using `Custom Tasks`, we need to define a [`Run`](https://github.com/tektoncd/pipeline/blob/master/docs/runs.md)
-type with `apiVersion: cel.tekton.dev/v1alpha1` and `kind: CEL`. The `Run` takes the CEL expressions to be evaluated
-as `Parameters`. If executed successfully, the `Run` will produce the evaluation results as `Results` with names corresponding
-with the `Parameters` names. See the [examples](examples) folder for `CEL` `Custom Tasks` to run or use as samples. 
+To evaluate a CEL expressions using `Custom Tasks`, we need to define a [`CustomRun`](https://github.com/tektoncd/pipeline/blob/main/docs/customruns.md)
+type with `apiVersion: cel.tekton.dev/v1alpha1` and `kind: CEL`. The `CustomRun` takes the CEL expressions to be evaluated
```

Comment:
`v1beta1`?

Reply:
Do you mean `apiVersion: cel.tekton.dev/v1alpha1` -> `apiVersion: cel.tekton.dev/v1beta1`? (It might be a good idea though !)

Reply:
Yep! Sorry for the confusion :) But this is just a NIT, not necessarily block this PR.